e2e/connection_test.go

```diff
@@ -157,6 +157,9 @@ func consumerConnectFlow(t *testing.T, tequilapi *tequilapi_client.Client, consu
 	assert.Equal(t, connectionStatus.SessionID, se.SessionID)
 	assert.Equal(t, "New", se.Status)
 
+	// Wait some time for session to collect stats.
+	time.Sleep(2 * time.Second)
```

Comment:
@vkuznecovas After this change NAT e2e test started to fail with error

```
core/connection/manager.go:468           > Payment error error="could not calculate amount to promise: could not request consumer data accountant: could not execute request: Get http://accountant:8889/api/v2/data/consumer/0x7ab7f1f3838b94ca5dd539c3c0836c76c6f9ee9d: net/http: request canceled (Client.Timeout exceeded while awaiting headers)"
```

I see that you are using new http client instance https://github.com/mysteriumnetwork/node/blob/master/cmd/di.go#L497 instead of di.HttpClient. In such case this http client is not reconnected as implemented in https://github.com/mysteriumnetwork/node/blob/master/cmd/di.go#L873

Do you need such low timeout or can I change it to use global HTTPClient ?( which will also work more stable)

Reply:
I think we can change the client.